recipes/tardis/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "tardis-sn" %}
+{% set version = "3.0.a1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  git_url: https://github.com/tardis-sn/tardis.git
```

Comment:
Conda-forge only [supports "released" versions](http://conda-forge.org/docs/orga/guidelines.html?highlight=released#intended-usage) and generally prefers [tarballs to git repositories](http://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos).

This should probably be something like:
```yaml
    url: https://github.com/tardis-sn/tardis/archive/v{{ version }}.tar.gz
    sha256: aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
```

Reply:
Ok, working on it.

Reply:
Maybe @wkerzendorf is planning to release 3.0a1 soon. I'll wait for his response.

Reply:
So you want me to tag a release? but we always need to make a PR to update the version ?

Reply:
okay - made a release